### PR TITLE
migrate ci-kubernetes-e2e-prow-canary to community cluster

### PR DESCRIPTION
**Pull request by reetasingh**
migrate job listed in https://github.com/kubernetes/test-infra/issues/31793
@ameukam

### CODE REVIEW

File: config/jobs/kubernetes/sig-testing/kubetest-canaries.yaml [hunk 1]
```diff
@@ -21,6 +22,13 @@ periodics:
       - --provider=gce
       - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
       - --timeout=65m
+      resources:
+        limits:
+          cpu: 2
+          memory: 4Gi
+        requests:
+          cpu: 2
+          memory: 4Gi
```
Comment (ameukam) [inline config/jobs/kubernetes/sig-testing/kubetest-canaries.yaml:31]:
```suggestion
          memory: 4Gi
```

**Comment by ameukam** [comment]
@reetasingh Can you drop this line:

```
 - --gcp-zone=us-west1-b
 ```
 ?

**Comment by reetasingh** [comment]
> @reetasingh Can you drop this line:
> 
> ```
>  - --gcp-zone=us-west1-b
> ```
> 
> ?

done

**Pull request merged**